src/store/entities.js

```diff
@@ -264,6 +265,17 @@ export default {
     [ACTION_TYPES.LOAD_TAGS_PENDING]: showTagsPending,
     [ACTION_TYPES.LOAD_TAGS_FULFILLED]: showTags,
     [ACTION_TYPES.ALL_TAGS_REJECTED]: loadingRejected,
+    [ACTION_TYPES.OPERATING_SYSTEMS_PENDING]: (state) => ({ ...state, operatingSystemsLoaded: false }),
+    [ACTION_TYPES.OPERATING_SYSTEMS_FULFILLED]: (state, { payload: { results } }) => ({
+        ...state,
+        operatingSystems: results.map(entry => {
+            const { name, major, minor } = entry.value;
+            const versionStringified = major.toString() + '.' + minor.toString();
```

Comment:
No need to call `toString`, we can use string literals
```suggestion
            const versionStringified = `${major}.${minor}`;
```